mlrun/model_monitoring/controller.py

```diff
@@ -426,22 +434,18 @@ def model_endpoint_process(
                         parquet_target_path = offline_response.vector.get_target_path()
 
                         if len(df) == 0:
-                            logger.warn(
-                                "Not enough model events since the beginning of the batch interval",
-                                featureset_name=m_fs.metadata.name,
+                            logger.info(
+                                "Empty window",
                                 endpoint=endpoint[mm_constants.EventFieldType.UID],
-                                min_required_events=mlrun.mlconf.model_endpoint_monitoring.parquet_batching_max_events,
                                 start_time=start_infer_time,
                                 end_time=end_infer_time,
                             )
                             continue
 
-                    # Continue if not enough events provided since the deployment of the model endpoint
                     except FileNotFoundError:
```

Comment:
As we see the get offline function creates parquet even if it is empty, so I think we need to catch it here (maybe the last catch is enough), and if we catching it we need to change the logger level to warn/error.

Reply:
d711f75